velox/buffer/tests/CMakeLists.txt

```diff
@@ -24,5 +24,5 @@ target_link_libraries(
   gtest_main
   gmock
   glog::glog
-  ${gflags_LIBRARIES}
```

Comment:
Using variables to link is not idiomatic for modern cmake and can cause issues (empty vars don't error but fail silently). This is basically a preview for the incoming cmake refactor :D